docs/UserGuide.md

```diff
@@ -284,13 +283,22 @@ logout
 
 </box>
 
-## Adding your first Delivery
+## Registering and creating your first Delivery
 
 Hopefully at this point, you have a good understanding of the basics of HomeBoss. Let's get you started on HomeBoss
-by adding your first Delivery
+by creating your first Delivery
+
+1. First register for HomeBoss using the `register`{.swift} command. So, for example, if you want to register an account
+   with the following details:
+
+> `USERNAME`: Alex123
+> `PASSWORD`: AlexIsGreat
+> `CONFIRM_PASSWORD`: AlexIsGreat
+> `SECRET_QUESTION`: First Pet Name? 
+> `ANSWER`: KoKo 
```

Comment:
Could you help me include `{.swift}` after the code block? Thank you : 3

Reply:
doneee